src/awsy.js

```diff
@@ -15,13 +15,6 @@ const DESKTOP_FIREFOX_APPS = {
     color: PALETTE.orange,
     project: ALT_PROJECT,
```

Comment:
Wouldn't it be necessary to add `extraOptions: ['fission'],` here to make fission default?

Reply:
Unfortunately, AWSY doesn't behave in the same was as Raptor. Whilst we had 'fission' in the extraOptions for a while, the default now has Fission enabled but without this in extraOptions.